packages/developer-portal/src/constants/urls.ts

```diff
@@ -0,0 +1,11 @@
+export const URLS = {
+  production: {
+    reapitConnectMyAccount: 'https://connect.reapit.cloud/my-account',
+  },
+  development: {
+    reapitConnectMyAccount: 'https://connect.dev.paas.reapit.cloud/my-account',
+  },
+  local: {
+    reapitConnectMyAccount: 'https://connect.dev.paas.reapit.cloud/my-account',
+  },
```

Comment:
because I don't know, might be worth checking beforehand, do these envs exist when deploying this application?